classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @student = []
```

Comment:
- [ ] This should be `@students` because in this class what you are always referencing is `students`, not `student`